system-addon/common/Reducers.jsm

```diff
@@ -329,11 +333,21 @@ function PreferencesPane(prevState = INITIAL_STATE.PreferencesPane, action) {
   }
 }
 
+function TippyTop(prevState = INITIAL_STATE.TippyTop, action) {
+  switch (action.type) {
+    case at.TIPPYTOP_INIT:
+    case at.TIPPYTOP_UPDATED:
+      return Object.assign({}, prevState, {initialized: true, sitesByDomain: action.data});
```

Comment:
@k88hudson should there be any concern about putting all the tippy top data into redux? Even if we didn't broadcast it on `TIPPYTOP_{INIT,UPDATED}`, it would make its way to content via rehydration. And it looks like there's no use of that data from content in this PR or near future (?) ?

Instead of putting tippytop data in the store, maybe we just stash it as some property on the feed?

Reply:
Good point. The way this ended up, we don't need this data in the redux store 👍